src/Kunstmaan/MediaBundle/Form/CroppableMediaLinkAdminType.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Kunstmaan\MediaBundle\Form;
+
+use Kunstmaan\MediaBundle\Entity\CroppableMediaLink;
+use Kunstmaan\MediaBundle\Form\Type\MediaType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CroppableMediaLinkAdminType extends AbstractType
```

Comment:
I think we should rename this formType to a more suitable name as it can do more than cropping